src/sagemaker/workflow/_utils.py

```diff
@@ -304,10 +283,10 @@ def __init__(
 
         Args:
             name (str): The name of the training step.
-            step_type (StepTypeEnum): The type of the step with value
-                `StepTypeEnum.Training`.
-            estimator (EstimatorBase): A `sagemaker.estimator.EstimatorBase` instance.
-            model_data: the S3 URI to the model data from training.
+            step_args (dict): The arguments for the `_RegisterModelStep` definition (default: None).
+            estimator (EstimatorBase): A `sagemaker.estimator.EstimatorBase` instance
+                (default: None).
+            model_data: the S3 URI to the model data from training (default: None).
```

Comment:
Can we improve the ordering of args in docstrings and maintain the order of fuction args.

Reply:
Sure, updated in the new commit